core/benchmarks/bm-nosonar_node_api.cpp

```diff
@@ -12,36 +12,31 @@
 #include <gnuradio-4.0/testing/bm_test_helper.hpp>
 
 inline constexpr std::size_t N_ITER = 10;
-// inline constexpr std::size_t N_SAMPLES = gr::util::round_up(1'000'000, 1024);
-inline constexpr std::size_t N_SAMPLES = gr::util::round_up(10'000, 1024);
+// inline constexpr gr::Size_t N_SAMPLES = gr::util::round_up(1'000'000, 1024);
+inline constexpr gr::Size_t N_SAMPLES = gr::util::round_up(10'000, 1024);
 
 template<typename T, char op>
 struct math_op : public gr::Block<math_op<T, op>, gr::PortInNamed<T, "in">, gr::PortOutNamed<T, "out">> {
-    T factor = static_cast<T>(1.0f);
-
-    // public:
-    //     math_op() = delete;
-    //
-    //     explicit math_op(T factor, std::string name = gr::this_source_location()) : factor(factor) { this->set_name(name); }
+    T value = static_cast<T>(1.0f);
 
     template<gr::meta::t_or_simd<T> V>
     [[nodiscard]] constexpr auto
     processOne(const V &a) const noexcept {
         if constexpr (op == '*') {
-            return a * factor;
+            return a * value;
         } else if constexpr (op == '/') {
-            return a / factor;
+            return a / value;
         } else if constexpr (op == '+') {
-            return a + factor;
+            return a + value;
         } else if constexpr (op == '-') {
-            return a - factor;
+            return a - value;
         } else {
```

Comment:
Is this block still needed? Can you use newly implemented Math block?

Reply:
removed this.